### ISSUE TITLE
Feedback:  First time using Warp

### ISSUE THREAD

**Issue by PaulFWatts** [warpdotdev/warp#548]
tldr: I like it!

Have finally gotten around to trying out Warp. Received an invite awhile back, but as they say "life happens".

I have been using the terminal and shells since that's all there was (started in computers in the mid 70's) 

I can see that Warp has a lot of potential for the following reasons:
1. It's fast (who doesn't love rust)
2. It's minimalist. Doesn't have a ton of features I will never use.
3. The features it does have I will use all the time and are well thought out
4. Easy to theme and build your own theme if you wish
5. Automatically configures the shell prompt depending where you are (e.g. in a directory with a git repo)
6. It will be cross-platform ( Looking forward to seeing the Windows, Linux and it running in the browser (WASM))
7. I'm used to the command palette shortcut key in a lot of my dev apps now. Great to see Warp has it. 

So, in general, great job guys. Look forward to further updates .....

**Comment by elviskahoro** [comment]
Thanks @PaulFWatts You can also email me directly with questions, feedback, and or feature requests:
elvis@warp.dev
